pod/import_video/views.py

```diff
@@ -589,9 +555,9 @@ def upload_youtube_recording_to_pod(request, record_id):
             "Try changing the access rights to the video directly in Youtube."
         )
         raise ValueError(msg)
-    except PytubeError:
+    except PytubeError as pterror:
         msg = {}
-        msg["error"] = _("YouTube error")
+        msg["error"] = _("YouTube error '%s'" % (mark_safe(pterror)))
```

Comment:
msg = {"error": _("YouTube error '%s'" % (mark_safe(pterror))), 
           "message": _(
        "YouTube content is inaccessible. "
        "This content does not appear to be publicly available."
    )}